[2025-06-23T17:01:07.937+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: DAG_atualizacao_desemenho_revenda.insert_data manual__2025-06-23T17:00:57.223592+00:00 [queued]>
[2025-06-23T17:01:07.954+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: DAG_atualizacao_desemenho_revenda.insert_data manual__2025-06-23T17:00:57.223592+00:00 [queued]>
[2025-06-23T17:01:07.954+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 1
[2025-06-23T17:01:08.002+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): insert_data> on 2025-06-23 17:00:57.223592+00:00
[2025-06-23T17:01:08.009+0000] {standard_task_runner.py:60} INFO - Started process 153474 to run task
[2025-06-23T17:01:08.015+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'DAG_atualizacao_desemenho_revenda', 'insert_data', 'manual__2025-06-23T17:00:57.223592+00:00', '--job-id', '114', '--raw', '--subdir', 'DAGS_FOLDER/DagAtualizacaoDesempenho.py', '--cfg-path', '/tmp/tmplj_u5wsq']
[2025-06-23T17:01:08.017+0000] {standard_task_runner.py:88} INFO - Job 114: Subtask insert_data
[2025-06-23T17:01:08.141+0000] {task_command.py:423} INFO - Running <TaskInstance: DAG_atualizacao_desemenho_revenda.insert_data manual__2025-06-23T17:00:57.223592+00:00 [running]> on host 99f5714845f7
[2025-06-23T17:01:08.380+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='DAG_atualizacao_desemenho_revenda' AIRFLOW_CTX_TASK_ID='insert_data' AIRFLOW_CTX_EXECUTION_DATE='2025-06-23T17:00:57.223592+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-23T17:00:57.223592+00:00'
[2025-06-23T17:01:08.486+0000] {base.py:83} INFO - Using connection ID 'dataset_comercial_database' for task execution.
[2025-06-23T17:01:36.936+0000] {logging_mixin.py:188} INFO - 202 registros inseridos com sucesso.
[2025-06-23T17:01:36.937+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-06-23T17:01:36.956+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=DAG_atualizacao_desemenho_revenda, task_id=insert_data, execution_date=20250623T170057, start_date=20250623T170107, end_date=20250623T170136
[2025-06-23T17:01:37.041+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-06-23T17:01:37.071+0000] {taskinstance.py:3280} INFO - 0 downstream tasks scheduled from follow-on schedule check
